spec.bs

```diff
@@ -786,9 +786,10 @@ An interest group is a [=struct=] with the following items:
 : <dfn>expiry</dfn>
 :: A point in time at which the browser will forget about this interest group.
 : <dfn>owner</dfn>
-:: An [=url/origin=].
+:: An [=url/origin=]. Frames that join interest groups owned by `owner` must either be served from origin `owner`,
+  or another origin delegated by `owner`. The scheme must be https.

```

Comment:
Try `The [=origin/scheme=] must be "<code>https</code>"`

Reply:
Done